app/Http/Controllers/ProjectController.php

```diff
@@ -96,15 +107,22 @@ public function edit($id)
      */
     public function update(Request $request, $id)
     {
-        $project = Project::find($id);
-        $project->name = $request->get('name');
-        $project->information = $request->get('information');
-        $project->deadline = $request->get('deadline');
-        $project->type = $request->get('type');
-        $project->status = $request->get('status');
-        $project->save();
-
-        return response()->json('Product Updated Successfully.');
+      $rules = [
+      'name' => 'required|unique:member,id,'.$id.'|regex:/^[a-zA-Z0-9-. ]+$/u|max:10',
```

Comment:
These rules are the same as above, so please use a class constant or property to prevent duplication